ansible/roles/package-upgrade/tasks/partials/brew.yml

```diff
@@ -36,10 +36,12 @@
     script: files/install-homebrew.sh
     when: os == "macos11.0" and not armbrew.stat.exists
 
-  - name: Upgrade installed packages
+  # Ignoring deprecation errors in MacOS 10.x
+  - name: Upgrade installed packages 
     become_user: "{{ ansible_user }}"
     homebrew:
       upgrade_all: yes
+    ignore_errors: os == "macos10.15" or os == "macos10.14"
```

Comment:
ansible tells me that the value is invalid:

`TASK [package-upgrade : Upgrade installed packages] ********************************************************************
fatal: [release-nearform-macos11.0-arm64-1]: FAILED! => {"msg": "the field 'ignore_errors' has an invalid value (os == \"macos10.15\" or os == \"macos10.14\"), and could not be converted to an bool.The error was: The value 'os == \"macos10.15\" or os == \"macos10.14\"' is not a valid boolean.  Valid booleans include: 'true', 1, 0, 'no', 'false', 'yes', 'n', 'y', '0', 'on', '1', 'off', 't', 'f'\n\nThe error appears to be in '/home/mzasso/git/nodejs/build/ansible/roles/package-upgrade/tasks/partials/brew.yml': line 40, column 5, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n  # Ignoring deprecation errors in MacOS 10.x\n  - name: Upgrade installed packages\n    ^ here\n"}
...ignoring`

Reply:
what version are you using? 🤔

Reply:
ansible [core 2.14.2]

Reply:
Does 
```suggestion
    ignore_errors: "{{ os == 'macos10.15' or os == 'macos10.14' }}"
```
work?

Reply:
yes

Reply:
I wonder if we should skip the step instead of ignoring errors. It takes ages to run because Homebrew doesn't support macOS <11 and it has to compile packages from source.

Reply:
> I wonder if we should skip the step instead of ignoring errors. It takes ages to run because Homebrew doesn't support macOS <11 and it has to compile packages from source.

The Ansible scripts are supposed to be able to both:
- Set up a new VM/machine from scratch.
- Update an existing VM/machine.

For Orka we could possibly get away with the first by relying on snapshots, but it wouldn't help for the Nearform VMs (I guess we could snapshot those but we'd have to find storage for the snapshots and we already have disk space issues on those machines).

I would suggest though that we plan to move off macOS 10.15, at least for Node.js 20 and beyond.